src/views/CoCurricularTranscript/guestTranscript.tsx

```diff
@@ -0,0 +1,116 @@
+import { MembershipHistory, MembershipHistorySession } from 'services/user';
+import { StudentProfileInfo } from 'services/user';
+import { Class } from 'services/peopleSearch';
+import { Participation } from 'services/membership';
+import { TranscriptItems } from 'services/transcript';
+
+const participation1: Participation = Participation.Member;
+const currentYear: string = new Date().getFullYear().toString();
+const session1: MembershipHistorySession = {
+  MembershipID: 2904,
+  SessionCode: currentYear + '01',
+  Participation: participation1,
+};
+const activity1: MembershipHistory = {
+  ActivityCode: 'ISO',
+  ActivityDescription: 'International Student Organization',
+  ActivityImagePath: '',
+  Sessions: [session1],
+  LatestDate: currentYear + '-05-10T00:00:00',
+};
+const participation2: Participation = Participation.Member;
+const session2: MembershipHistorySession = {
+  MembershipID: 6547,
+  SessionCode: currentYear + '09',
+  Participation: participation2,
+};
+const activity2: MembershipHistory = {
+  ActivityCode: '360',
+  ActivityDescription: '360.gordon.edu',
+  ActivityImagePath: '',
+  Sessions: [session2],
+  LatestDate: currentYear + '-12-15T00:00:00',
+};
+export const transcriptItems1: TranscriptItems = {
+  honors: [],
+  experiences: [],
+  service: [],
+  activities: [activity1, activity2],
+};
+export const student1: StudentProfileInfo = {
+  fullName: 'First M Last',
+  Majors: ['Undesided'],
+  Minors: [],
+  Advisors: [],
+  ID: '50232456',
```

Comment:
```suggestion
  ID: '12345678',
```
I'm guessing this is your ID number. You shouldn't make that public and it's unfortunate that it's now in the permanent GitHub history if so.

Reply:
It is not my ID but thank you! And I think 12345678 does look better as a mocked up ID.